entropylab/results_backend/sqlalchemy/tests/test_db.py

```diff
@@ -13,25 +13,57 @@ def test_save_result_raises_when_same_result_saved_twice(initialized_project_dir
         db.save_result(0, raw_result)
 
 
-def test_get_last_result_of_experiment_works_with_storage(initialized_project_dir_path):
+def test_get_last_result_of_experiment_when_hdf_is_enabled_then_result_is_from_hdf5(
+    initialized_project_dir_path,
+):
     # arrange
     db = SqlAlchemyDB(initialized_project_dir_path, enable_hdf5_storage=False)
-    db.save_result(1, RawResultData(label="save", data="to db"))
+    db.save_result(1, RawResultData(label="save", data="in db"))
     db = SqlAlchemyDB(initialized_project_dir_path, enable_hdf5_storage=True)
-    db.save_result(1, RawResultData(label="save", data="to storage"))
+    db.save_result(1, RawResultData(label="save", data="in storage"))
     # act
     actual = db.get_last_result_of_experiment(1)
     # assert
-    assert actual.data == "to storage"
+    assert actual.data == "in storage"
 
 
-def test_get_last_result_of_experiment_works_with_db(initialized_project_dir_path):
+def test_get_last_result_of_experiment_when_hdf_is_disabled_then_result_is_from_db(
+    initialized_project_dir_path,
+):
     # arrange
     db = SqlAlchemyDB(initialized_project_dir_path, enable_hdf5_storage=True)
-    db.save_result(1, RawResultData(label="save", data="to storage"))
+    db.save_result(1, RawResultData(label="save", data="in storage"))
     db = SqlAlchemyDB(initialized_project_dir_path, enable_hdf5_storage=False)
-    db.save_result(1, RawResultData(label="save", data="to db"))
+    db.save_result(1, RawResultData(label="save", data="in db"))
     # act
     actual = db.get_last_result_of_experiment(1)
     # assert
-    assert actual.data == "to db"
+    assert actual.data == "in db"
+
+
+# def test_save_figure_(initialized_project_dir_path):
```

Comment:
can you delete the commented tests?

Reply:
One deleted, one un-commented and fixed. 

Would love to have this merged unless there are other issues?